spec/lucky_flow_spec.cr

```diff
@@ -7,7 +7,7 @@ describe LuckyFlow do
 
     flow.visit("/home")
 
-    flow.el("@heading", text: "Home").should be_on_page
```

Comment:
Maybe we can just leave this as well and it also serves as documentation when people go digging around the specs?